### PR TITLE
ACC-2235: create new liveChat flag to enable or disable chat

### PR DESCRIPTION
**Pull request by danieleruiz**
### Description
The team that runs the liveChat is doing a big migration soon! They want to have a flag to turn off the liveChat over a few hours so they can ensure no real users connect to the chat, even though some agents will be online testing things.
### Ticket
[ACC-2235](https://financialtimes.atlassian.net/browse/ACC-2235)

[ACC-2235]: https://financialtimes.atlassian.net/browse/ACC-2235?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/js/browser.ts [hunk 1]
```diff
@@ -53,55 +53,59 @@ class LiveChat {
...
+					};
+					setTimeout(initLiveChat, displayDelay > 1000 ? displayDelay : 1000);
 				};
-				setTimeout(initLiveChat, displayDelay > 1000 ? displayDelay : 1000);
-			};
-			document.head.appendChild(script);
+				document.head.appendChild(script);
+			}
+		} else {
+			return ()=>{}
```
Comment (Nazehs) [inline src/js/browser.ts:108]: [Comment] - Is this an intentional act that we are returning an empty object when the live chat flag is not on?

Comment (danieleruiz) [reply src/js/browser.ts:108]: the initializer function has defined a Function return type so i just return an empty function when i don´t want to initiate the livechat

Comment (Nazehs) [reply src/js/browser.ts:108]: [Comment] I thought the they do want to initialise the livechat with their staging url for testing or the livechat would be initialise with prod url. That's my assumption. If this is the right assumption it means the livechat can only be initialise if the toggle the flag unless product can work without the initialising. I might be wrong anyway

Comment (binaryberry) [reply src/js/browser.ts:108]: @Nazehs what you're talking about is a thing, but this ticket is a new request to turn off live chat completely while they are testing a migration

Comment (Nazehs) [reply src/js/browser.ts:108]: Ooh okay now i get it. thank you  @binaryberry  i am happy to approve it

**Review by Nazehs — APPROVED** [review]

**Pull request merged**